src/msg/async/AsyncConnection.cc

```diff
@@ -138,7 +138,6 @@ AsyncConnection::AsyncConnection(CephContext *cct, AsyncMessenger *m, DispatchQu
   write_handler = new C_handle_write(this);
   wakeup_handler = new C_time_wakeup(this);
   tick_handler = new C_tick_wakeup(this);
-  memset(msgvec, 0, sizeof(msgvec));
```

Comment:
I don't like this cleaning, memset call isn't expensive in this path and zeroed memory will be good for other debug purpose

Reply:
this parameter don,t use anymore now.